script/upgrades/dependencies.json

```diff
@@ -1,5 +1,9 @@
 {
-  "42220": {},
+  "42220": {
+    "BridgedUSDC": "0xEB466342C4d449BC9f53A865D5Cb90586f405215",
+    "USDCUSDRateFeedAddr": "0xA1A8003936862E7a15092A91898D69fa8bCE290c",
```

Comment:
what is this EOA address?

Reply:
This is the RateFeedID of the USDCUSD oracle rate. It's the target address used for reporting. It's that `address(uint160(keccak256("USDCUSD")))` thing that we couldn't find a good place to put in the docs 😂

Reply:
aah now it makes sense :D maybe call it `...RateFeedID` instead of `RateFeedAddr`?